test/k8sT/manifests/kafka-sw-security-policy.yaml

```diff
@@ -4,6 +4,17 @@ description: "Allow only permitted Kafka requests to empire Kafka broker"
 metadata:
   name: "kafka-sw-security-policy"
 specs:
+  - endpointSelector:
```

Comment:
When we run Cilium alongside Kubernetes, we automatically allow ingress from the host to allow for healthchecks / liveness probes to succeed. This is most likely not necessary, but I don't see an issue with explicitly allowing access from the host at the specific port. Have you seen policy denying ingress from the host in the Kafka CI tests? If so, please provide examples where this is the case.